server.js

```diff
@@ -0,0 +1,28 @@
+const helmet = require("helmet");
+const logger = require("morgan");
+const express = require("express");
+const cors = require("cors");
+const users = require("./users/userRouter");
+const posts = require("./posts/postRouter");
+const server = express();
+
+//Builtin MiddleWare
+server.use(express.json());
+
+// Third Party MiddleWare
+server.use(cors());
+server.use(helmet());
+server.use(logger("tiny"));
+
+//Routes
+server.use("/api/users/", users);
+server.use("/api/posts/", posts);
+
+server.get("/", (req, res) => {
+  const motd = process.env.MOTD || "You are not Prepared!";
```

Comment:
I'm curious what problem this solves as your front end will never purposely go to this route and I don't think there is any security risks this would cover up. In fact I think that the only thing you hope to gain by doing this would be having to add another env var to your deployment for your server to work properly.